README.md

```diff
@@ -511,11 +504,44 @@ In order to use environment variable in deployment or cronjob, you will have to
    you can either provide `nameSuffix` which means name added after prefix ```<applicationName>-``` or static name with ```name``` of secret
 
    **Note:** first key after ``envFrom`` is just used to uniquely identify different objects in ``envFrom`` block. Make sure to keep it unique and relevant 
+
+
+### Configuring probes
+
+By default probe handler type is `httpGet`. You just need to override `port` and `path` as per your need.
+
+```
+  livenessProbe:
+    httpGet:
+      path: '/path'
+      port: 8080
+```
+
+
+In order to use `exec` handler, you can define field `livenessProbe.exec` in your values.yaml.
+
+```
+  livenessProbe:
+    exec:
+      command:
+        - cat
+        - /tmp/healthy
+```
+
+To disable liveness or readiness probe, set value of `enabled:` to `false`.
+```
+  livenessProbe:
+    enabled: false
+```
+
+
  
 # Changelog
 
 All notable changes to this project will be documented here
 
+## v1.1.9
+- Feature: add functionality to disable liveness and readiness probes.
```

Comment:
isn't it more than that? we can also set individual values on the probes?

Reply:
previously we could set on full block and not individual values of the probes; plz double check